File: pinyougou-parent/pinyougou-interface/src/main/java/cn/itcast/core/service/seckillGoodsService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.good.Goods;
import cn.itcast.core.pojo.seckill.SeckillGoods;
import entity.PageResult;
import vo.GoodsVo;

public interface seckillGoodsService {
    /*void add(GoodsVo goodsVo);*/

    PageResult search(Integer page, Integer rows, SeckillGoods seckillGoods);

    /*GoodsVo findOne(Long id);

    void update(GoodsVo goodsVo);*/

    void updateStatus(Long[] ids, String status);

    /*void delete(Long[] ids);*/
    SeckillGoods findOne(Long id );

    void add(SeckillGoods seckillGoods);

    void update(SeckillGoods seckillGoods);


    PageResult search1(Integer page, Integer rows, SeckillGoods seckillGoods);
    void dele(Long[] ids);

}

File: pinyougou-parent/pinyougou-web-shop/src/main/webapp/js/service/seckill_goodsService.js
//服务层
app.service('seckill_goodsService',function($http){

    //查询实体
    this.findOne=function(id){
        return $http.get('../SeckillGoods/findOne.do?id='+id);
    }
    //增加
    this.add=function(entity){
        return  $http.post('../SeckillGoods/add.do',entity );
    }
    //修改
    this.update=function(entity){
        return  $http.post('../SeckillGoods/update.do',entity );
    }
    //删除
    this.dele=function(ids){
        return $http.get('../SeckillGoods/delete.do?ids='+ids);
    }
    //搜索
    this.search=function(page,rows,searchEntity){
        return $http.post('../SeckillGoods/search.do?page='+page+"&rows="+rows, searchEntity);
    }
});
